hotsos/core/issues/__init__.py

```diff
@@ -1,5 +1,10 @@
-from .issue_types import *  # noqa: F403,F401
-from .utils import (   # noqa: F403,F401
+from .issue_types import *  # noqa:F403,F401
```

Comment:
Don't think missing space after noqa: matters

Reply:
ack that was unintentional will fix